setupsrc/pypdfium2_setup/build_native.py

```diff
@@ -171,19 +171,20 @@ def get_sources(deps_info, short_ver, with_tests, compiler, clang_path, single_l
             r'(\s*)("//third_party/test_fonts")', r"\1# \2",
             is_regex=True, exp_count=1,
         )
-        if single_lib:
-            autopatch(
-                PDFIUM_DIR/"BUILD.gn",
-                'component("pdfium")',
-                'shared_library("pdfium")',
-                is_regex=False, exp_count=1,
-            )
-            autopatch(
-                PDFIUM_DIR/"public"/"fpdfview.h",
-                "#if defined(COMPONENT_BUILD)",
-                "#if 1  // defined(COMPONENT_BUILD)",
-                is_regex=False, exp_count=1,
-            )
+        # bundle dependencies (e.g. abseil) into the pdfium DLL
+        # TODO in the future, we might want to extract separate DLLs for the imaging libraries (e.g. libjpeg, libpng)
```

Comment:
I just realized this comment is out of place for the native sourcebuild, given that use_system_libjpeg and so on are set to true. This needs to be moved to the toolchained sourcebuild instead.

Reply:
Done in #377.